SparseGrids/TasmanianSparseGrid.cpp

```diff
@@ -1585,6 +1585,14 @@ void tsgMakeLocalPolynomialGrid(void *grid, int dimensions, int outputs, int dep
 void tsgMakeWaveletGrid(void *grid, int dimensions, int outputs, int depth, int order, const int *limit_levels){
     ((TasmanianSparseGrid*) grid)->makeWaveletGrid(dimensions, outputs, depth, order, limit_levels);
 }
+void tsgMakeFourierGrid(void *grid, int dimensions, int outputs, int depth, const char *sType, const int *anisotropic_weights, const int *limit_levels){
```

Comment:
* the C interface is good, but must test the grid I/O as well
* inside `testsTSG.in.py`, look for `def testBasicIO()`, between the tests for wavelets and conformal transforms add a few tests regarding the read/write in different formats